services/filemanager/filemanager.go

```diff
@@ -14,6 +14,9 @@ var (
 	DefaultFileManagerFactory FileManagerFactory
 )
 
+const (
+	InvalidServiceProvider = "service provider not supported"
```

Comment:
Make common error package and use it everywhere.

Reply:
Probably a more descriptive error message...  If its an empty string it's not configured else it isn't supported?

Reply:
done